### ISSUE TITLE
Wrong postgres diff with GenerationType.AUTO

### ISSUE THREAD

**Issue by cbornet** [liquibase/liquibase-hibernate#105]
When GenerationType.AUTO is used with postgres, the hibernate_sequence is used so PK shouldn't have auto_increment.
* If the table was generated with auto_increment=true, then a diff wants to remove the sequence (but not the PK default value using it !)
* If the table was generated with auto_increment=false, then a diff wants to add the autoincrement

Example of diff
```xml
  <changeSet author="vagrant (generated)" id="1453890642070-15">
      <dropSequence sequenceName="foo_id_seq"/>
   </changeSet>
   <changeSet author="vagrant (generated)" id="1453890642070-16">
      <addDefaultValue columnDataType="bigint"
                       columnName="id"
                       defaultValueComputed="nextval('bar_id_seq'::regclass)"
                       tableName="bar"/>
      <addAutoIncrement columnDataType="bigint" columnName="id" tableName="bar"/>
   </changeSet>
```
where foo was generated with auto_increment=true and bar was generated with auto_increment=false.

**Comment by cbornet** [comment]
Maybe related to #78

**Comment by nvoxland** [comment]
Fixed with liquibase-hibernate 3.6 and liquibase 3.5.2
